src/components/Pool/PairView/UserLiquidity/UserLiquidity.container.tsx

```diff
@@ -0,0 +1,99 @@
+import { JSBI, Percent, TokenAmount } from '@swapr/sdk'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import { Box, Flex, Text } from 'rebass'
+
+import { useTotalSupply } from '../../../../data/TotalSupply'
+import { useActiveWeb3React } from '../../../../hooks'
+import { useTokenBalance } from '../../../../state/wallet/hooks'
+import { currencyId } from '../../../../utils/currencyId'
+import { unwrappedToken } from '../../../../utils/wrappedCurrency'
+import { UserLiquidityProps } from './UserLiquidity'
+import { ButtonExternalLink, ButtonPurpleDim } from '../../../Button'
+import { DimBlurBgBox } from '../../DimBlurBgBox'
+import { ValueWithLabel } from '../ValueWithLabel/ValueWithLabel.component'
+
+export function UserLiquidity({ pair }: UserLiquidityProps) {
+  const { account, chainId } = useActiveWeb3React()
+  const currency0 = unwrappedToken(pair?.token0)
+  const currency1 = unwrappedToken(pair?.token1)
+  const userPoolBalance = useTokenBalance(account ?? undefined, pair?.liquidityToken)
+  const totalPoolTokens = useTotalSupply(pair?.liquidityToken)
+  const { t } = useTranslation()
+
+  const accountAnalyticsLink = account
+    ? `https://dxstats.eth.link/#/account/${account}?chainId=${chainId}`
+    : `https://dxstats.eth.link/#/accounts?chainId=${chainId}`
```

Comment:
I see this pattern with account and chainId being used 8 times I think it makes sense to make a function out of it similar to how we did explorer one...

Reply:
Yes, we can do that. Will fix.